paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt

```diff
@@ -131,6 +140,69 @@ internal class PaymentSheetPlaygroundActivity : AppCompatActivity() {
         disableViews()
     }
 
+    override fun onResume() {
+        super.onResume()
+        val sharedPreferences = getSharedPreferences(sharedPreferencesName, MODE_PRIVATE)
```

Comment:
The fetching and reading of sharedpreferences should live in the model rather than the view. See https://developer.android.com/jetpack/guide?gclid=Cj0KCQiAraSPBhDuARIsAM3Js4pN8c0e-ouiwOFPtLguuwTTwKkJonnjQRfRnEiRHPMOeU2PlSbC5Z0aApKbEALw_wcB&gclsrc=aw.ds